### PR TITLE
Fix: Fixed an issue where Run with PowerShell didn't show a window

### LINKED ISSUE #15826: Bug: Run with PowerShell toolbar button doesn't show PowerShell window
### Description

After selecting a file, specifically a PowerShell Script, from the listings and then clicking the button "Run with PowerShell" it doesn't do anything. No error messages or anything. @Josh65-2201 confirms it doesn't for them either too.

### Steps To Reproduce

1. Select a .ps1 file
2. Click "Run with PowerShell" at the top

### Requirements

PowerShell should open with the selected script and execute it

### Files Version

3.5.0.0

### Windows Version

10.0.19045.4651

### Log File

[debug.log](https://github.com/user-attachments/files/16315601/debug.log)

### PR DESCRIPTION
**Pull request by Qin-shihuang**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #15826

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Open Files 
2. Select a Powershell Script (.ps1)
3. Cilck the "Run with PowerShell" button on the toolbar.
4. Before: The script executes in the background, without displaying a window (the script will stuck if it need to interact with the user).
	After:  The script executes in a new window.

### CODE REVIEW

**Comment by Qin-shihuang** [comment]
@yaira2 That's a good idea. Should I place it in `Files.App.Data.Enums`?

**Comment by yaira2** [comment]
Yes

File: src/Files.App/Data/Enums/PowerShellOptions.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	/// Represents options for PowerShell execution.
+	/// </summary>
+	[Flags]
+	public enum PowerShellOptions
+	{
+		/// <summary>
+		/// Default options.
+		/// </summary>
+		None = 0x0,
+		/// <summary>
```
Comment (0x5bfa) [inline src/Files.App/Data/Enums/PowerShellOptions.cs:16]:
```suggestion

		/// <summary>
```

File: src/Files.App/Data/Enums/PowerShellOptions.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		/// Run the PowerShell command hidden.
+		/// </summary>
+		Hidden = 0x1,
+		/// <summary>
```
Comment (0x5bfa) [inline src/Files.App/Data/Enums/PowerShellOptions.cs:20]:
```suggestion

		/// <summary>
```

**Review by 0x5bfa — APPROVED** [review]
LGTM code wise.
But please rename the enum to `PowerShellExecutionOptions`

**Review by 0x5bfa — APPROVED** [review]

**Pull request merged**